### DISCUSSION TITLE
Keep xDrip+ Follower's predictive settings updated

### DISCUSSION THREAD

**Discussion by mwarrildt** [NightscoutFoundation/xDrip#2947] [Q&A]
Hi
xDrip+ Followers get copied on cgm readings but (AFAIK) not on changes made to the master's Predictive Settings.

My wife uses xDrip+ Follower to keep an eye on me.  But I tweak my Predictive Settings (carb ratios) a lot (honeymoon from Hell) so her app isn't always using the current settings.

Is there a switch we can flip so that the Follower will also track Predictive settings?

Thanks
Mark

**Comment by Navid200** [comment]
I guess I understand what you want.  But, I'm not sure it is the best path forward.

I could be wrong.  So, please correct me.

I have diabetes and use xDrip.  However, I don't use predictive simulation.

When someone uses xDrip as their CGM, they could use it to decide how much insulin to take.  They must be making that decision on their own because the xDrip official position is that it should not be used for making dosing decisions.

If someone uses xDrip as their CGM and they choose to decide how much insulin to take, they may decide to use predictive simulations.

But, when someone uses xDrip as a follower, they do it to see what the glucose level of the person they are following is.
I don't see myself wanting to see predictive simulations for someone I follow if I was following someone.

As an example, you may enter insulin you have taken and eat but forget to enter carbs you have taken.
Your follower could get really worried if they could see your predictions.  While looking at your glucose, they would have no reason to worry.

This is not scientific explanation.  I'm just trying to understand how practical this would be and I have a hard time convincing myself that this is something we need.

**Comment by mwarrildt** [comment]
Thank you for your thoughts and the issues informing your conclusion.

Regarding dosing decisions, I do use my own calculations, but vet my
decision against Omnipod's and then xDrip's suggestions, expecting the
three independent answers to be similar.

In my specific case, my wife and I are the only ones involved, and while we
both understand the predictions to be guesses and by nature inaccurate and
unreliable, they still offer a measure of comfort.  The combination of
blood glucose and predictions have occasionally been useful, allowing her
to come get my attention when I've been busy and neglecting to track my bg.

I understand the team has to move carefully.  I'm glad you do.  Thanks
again for your thoughts.

Mark
